src/styles/catalog.scss

```
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $card-width;
```

why you didn't specify  generic parameters? for example:

```suggestion
  grid-template-columns: repeat(4, 1fr);
```